src/middleware/validateRating.js

```diff
@@ -0,0 +1,32 @@
+import Joi from 'joi';
+
+export default {
```

Comment:
Nice implementation, could you check the way the validations was implemented in the [validationSchema.js](https://github.com/andela/ah-kifaru-backend/blob/develop/src/middleware/validation.schema.js) module?, I do not think you need another module for this. I think this would help us maintain consistency in our codebase